tests/Migration/MigrationCreatorTest.php

```diff
@@ -22,6 +22,12 @@ public function testCreate(): void
     {
         $migrationCreator = new MigrationCreator('\Abc\Def', '    ');
         $migrationDir = __DIR__ . '/temp';
+        if (!is_dir($migrationDir)) {
```

Comment:
This is not very good test, you are not testing anything. Try. to add some configuration where there is no existing folder and after test it exists and contains file with migratino